fluster/main.py

```diff
@@ -48,22 +49,58 @@ class Main:
     def __init__(self) -> None:
         self.decoders_dir = DECODERS_DIR
         self.test_suites_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), TEST_SUITES_DIR
-        )
-        # Only use the system directory for test suites if the local directory
-        # doesn't exist and the system directory does exist.
-        if (
-            sys.platform.startswith("linux")
-            and not os.path.exists(self.test_suites_dir)
-            and os.path.exists(TEST_SUITES_DIR_SYS)
-        ):
-            self.test_suites_dir = TEST_SUITES_DIR_SYS
-
+            os.path.dirname(__file__), "..", TEST_SUITES_DIR
+        )
         self.resources_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESOURCES_DIR
+            os.path.dirname(__file__), "..", RESOURCES_DIR
         )
         self.results_dir = os.path.join(gettempdir(), RESULTS_DIR)
 
+        is_uninstalled = os.path.exists(
+            os.path.join(os.path.dirname(__file__), "..", ".git")
+        )
+        if not is_uninstalled:
+            appname = "fluster"
+            site_data_dir = platformdirs.site_data_dir(appname)
+            user_data_dir = platformdirs.user_data_dir(appname)
+
+            test_suites_dir = os.path.join(site_data_dir, TEST_SUITES_DIR)
+            if os.path.exists(test_suites_dir):
+                self.test_suites_dir = test_suites_dir
+            else:
+                self.test_suites_dir = os.path.join(user_data_dir, TEST_SUITES_DIR)
+
+            resources_dir = os.path.join(site_data_dir, RESOURCES_DIR)
+            if os.path.exists(resources_dir):
+                self.resources_dir = resources_dir
+            else:
+                self.resources_dir = os.path.join(user_data_dir, RESOURCES_DIR)
+
+            print(
```

Comment:
how about just printing where `test_suites_dir` and `resources_dir` is in both cases (e.g. outside of the `is_installed` if statement) to de-duplicate code ?

Reply:
Fixed in 720b1a5